client/src/main/java/com/alibaba/nacos/client/naming/remote/http/NamingHttpClientProxy.java

```diff
@@ -320,6 +320,7 @@ public boolean serverHealthy() {
             String serverStatus = json.get("status").asText();
             return "UP".equals(serverStatus);
         } catch (Exception e) {
+            NAMING_LOGGER.warn("[serverHealthy] Failed to get server health status");
```

Comment:
Why remove this log？

Reply:
this is add log，not remove。

Reply:
sorry, my mistake, but should we print the exception info?

Reply:
i think when request failed，need to print some info，not just return false.

Reply:
I know, I means print more information for exception detail

Reply:
oh , yes，need to print exception